src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -4073,6 +4078,8 @@ def _is_msi_cluster(managed_cluster):
 
 def _get_kubelet_config(file_path):
     kubelet_config = get_file_json(file_path)
+    if not os.path.isfile(kubelet_config):
```

Comment:
kubelet_config should be one Dict.
so the os.path.isfile should always return false?

Reply:
Oh actually that's right, good catch